stacks-in-depth/nodes-and-miners/mine-mainnet-stacks-tokens.md

```diff
@@ -49,20 +49,15 @@ txindex=1
 Finally, start bitcoin as follows (adjust the `conf` path to where it was created in the previous step, i.e. `$HOME/bitcoin.conf`):
 
 ```bash
-bitcoind -conf=$HOME/bitcoin.conf
+bitcoind  or bitcoin-qt
```

Comment:
no. `bitcoin-qt` is nothing sometihng we'll be adding to these docs.

Reply:
really works either way using bitcoind or bitcoin-qt they both work 100% using these instructions for people that like using a GUI

Reply:
nevertheless, these docs are not written with a gui in mind. remove mention of bitcoin-qt